filters/VoxelCentroidNearestNeighborFilter.cpp

```diff
@@ -71,8 +70,6 @@ PointViewSet VoxelCentroidNearestNeighborFilter::run(PointViewPtr view)
     BOX3D bounds;
     calculateBounds(*view, bounds);
```

Comment:
calculateBounds makes a full pass through the data.  I think this can be avoided since it seems like we're just using the min x/y/z as the origin for the voxel space.  I think using the value of the first point would do just as well if the r/c/d calculation were modified a bit.